components/login/__snapshots__/index.test.jsx.snap

```diff
@@ -16,22 +16,74 @@ exports[`Login form renders a login page with a sign in button and a 'Sign in wi
       src="/pod-spaces-logo.svg"
       width="160"
     />
+    <form>
```

Comment:
Added the wrapping form because something weird seemed to be happening without it, it may not be needed